### PR TITLE
Submodule update, refs #0

### PR DESCRIPTION
**Pull request by GiudGiud**
This will facilitate a few of the ongoing PRs

### CODE REVIEW

**Comment by YaqiWang** [comment]
This can be merged?

**Comment by GiudGiud** [comment]
No. Look at the tests. 
I m working on it though

**Comment by YaqiWang** [comment]
Oh, they are failed but allowed...

File: htgr/pbmr400/steady/tests [hunk 1]
```diff
@@ -94,15 +94,16 @@
...
     executable_pattern = 'dire_wolf*|blue_crab*|fenix*'
     # This is not a lot of iterations, because the model is too slow and because the convergence criteria
     # are set very loose which causes convergence issues on a second step
     cli_args = "Executioner/fixed_point_max_its=1 Executioner/num_steps=1 "
                "MultiApps/th_sub/cli_args='Executioner/num_steps=12' "
-               "MultiApps/th_sub/max_procs_per_app=1"
+               "MultiApps/th_sub/max_procs_per_app=1 Outputs/file_base=core"
     method = 'opt' # too slow for devel
     # Checkpoint file must have been generated with same number of ranks
     prereq = 'steady_multiphysics'
```
Comment (GiudGiud) [inline htgr/pbmr400/steady/tests:109]:
```suggestion
    prereq = 'steady_multiphysics'
    # Inconsistency between gold on macbook and test machine
    rel_err = 2e-5
```

**Comment by GiudGiud** [comment]
Test failures are inconsistences in griffin treatment of chi!=1 in opt/devel
and some new gFHR model failures I ll address separately

**Pull request merged**